Simplenote/SPAppDelegate.m

```diff
@@ -325,6 +326,7 @@ - (void)onboardingDidFinish:(NSNotification *)notification
 
 - (void)applicationWillResignActive:(UIApplication *)application
 {
+    [self.tagListViewController removeKeyboardObservers];
```

Comment:
Instead of exposing this method in the header of the tag list view controller and calling it from here, the tag list view controller could register for the system `UIApplicationWillResignActiveNotification` notification and call the method itself. Then the app delegate doesn't need any knowledge of the tag list view controller.